assets/scss/init/_variables.scss

```diff
@@ -14,7 +14,7 @@ $big-z-index: 1000;
 $font-family-base:              -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
 $headings-font-family:          'Source Sans Variable', sans-serif;
 
-$font-size-base:                0.9rem;
+$font-size-base:                1rem;
```

Comment:
Not sure if I like this. If we do re-set the base font size to 1 rem, we should also review line-heights and paddings here and there to keep it balanced.

Reply:
This was step 1 in a redo to come. I would like to set the base to 1rem/16px (assumed browser default) and work from there to the rest.

Reply:
OK. The common js is now in common.js